model.py

```diff
@@ -73,7 +73,7 @@ class Sighting(ModelMixin, db.Model):
     sighting_id = db.Column(db.Integer, autoincrement=True, primary_key=True)
     user_id = db.Column(db.Integer, db.ForeignKey('users.user_id'))
     # If I set pokemon_id to unique, will it allow only one of each for the whole db?
-    pokemon_id = db.Column(db.Integer, db.ForeignKey('pokemon.pokemon_id'))
+    pokemon_id = db.Column(db.Integer, db.ForeignKey('pokemon.pokemon_id'), unique=True)
```

Comment:
So this is only suggested if the `Sightings` only cares about unique sightings. If this keeps track of all sightings regardless, then ignore.

Reply:
Tested, prevents other users from logging sightings of the same species which is not what I want my app to do. I want to restrict an individual user from logging multiple entries of the same species.